mantle/platform/conf/conf.go

```diff
@@ -332,6 +374,27 @@ func (c *Conf) MergeV34exp(newConfig v34exptypes.Config) {
 	c.ignitionV34exp = &mergeConfig
 }
 
+// Merge all configs into a V3.4 config
+// configs must be of type V34exp to merge
+func MergeUserData(confObjs []*Conf) (*UserData, error) {
```

Comment:
Naming seems misleading here since we're merging configs. Why not `MergeV34exp`? That way it also communicates that they must be `v3.4-experimental` configs.